content/v2.0-preview/_index.md

```diff
@@ -1,51 +1,53 @@
 ---
-title: "Overview"
+title: "Welcome"
```

Comment:
i feel like this page is missing an end to end practical example to tie everything together - e.g. a user comes to a control plane built by the platform team with crossplane and uses it to deploy their application along with everything it needs. should that big picture practical value be conveyed on this page somehow?

Reply:
I'm not sure I agree.

I agree we need an example like that, but I'm not sure this landing page is the right place for it. I prefer to keep it relatively short and sweet - but hopefully enough that folks'll click through to other pages to learn more.

If you look at https://helm.sh/docs/ or https://kubernetes.io/docs/home/ for example, they only _very_ briefly mention what the actual project is.

Reply:
I could imagine us having a distinct "Why Crossplane" page similar to https://kubernetes.io/docs/concepts/overview/#why-you-need-kubernetes-and-what-can-it-do

Reply:
I ended up refactoring this into a new "What is Crossplane?" page. It now has practical examples and diagrams.